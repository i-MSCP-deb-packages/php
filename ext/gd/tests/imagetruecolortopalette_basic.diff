002+ 084e28df0d2906d48e2004450235192d
002- 0843f63ab2f9fddedd69b0b421686bc5