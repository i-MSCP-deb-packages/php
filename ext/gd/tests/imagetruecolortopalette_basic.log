
---- EXPECTED OUTPUT
bool(true)
0843f63ab2f9fddedd69b0b421686bc5
---- ACTUAL OUTPUT
bool(true)
084e28df0d2906d48e2004450235192d
---- FAILED
