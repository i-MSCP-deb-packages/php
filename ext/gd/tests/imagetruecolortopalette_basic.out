bool(true)
084e28df0d2906d48e2004450235192d